src/hub.html

```diff
@@ -183,13 +160,75 @@
                     class="interactable"
                     super-networked-interactable="counter: #counter; mass: 1;"
                     body="type: dynamic; shape: none; mass: 1;"
+                    auto-scale-cannon-physics-body
                     grabbable
-                    stretchable="useWorldPosition: true;"
+                    stretchable="useWorldPosition: true; usePhysics: never"
```

Comment:
why `usePhysics: never`?

Reply:
The `stretchable` component's adjustment of the cannon body's shapes' scale is redundant with `auto-scale-cannon-physics-body`, but cannot be used at the same time because of the way it calculates the desired scale (It does something like current (cannon shape) scale * delta, instead of current (entity) scale * shape component's half-extents or radius).
Stretchable thus changes the scales of the objects in cannon land but those changes don't get networked. I wrote `auto-scale-cannon-physics-body` to solve this problem by telling super-hands to only adjust the entity's scale and then change the cannon shapes' scale independently (and on both ends of the network).

Reply:
👍